docs/tutorials/dotnet-systemd.md

```diff
@@ -1,74 +1,88 @@
-# Run a .Net Echo Bot as a systemd service on Ubuntu WSL
+# Run a .NET Echo Bot as a systemd service on Ubuntu WSL
 *Authored by Oliver Smith ([oliver.smith@canonical.com](mailto:oliver.smith@canonical.com))*
 
-.Net is an open-source development platform from Microsoft that enables developers to build multi-platform applications from a single codebase.
+In this tutorial we will take advantage of WSL's systemd support to run a chatbot as a systemd service for easier deployment.
 
-In this tutorial we demonstrate how easy it is to start working with .Net on Ubuntu WSL by creating a simple chatbot accessible from your Windows host. We also take advantage of WSL's new systemd support to run our bot as a systemd service for easier deployment.
+We will create the bot using .NET on Ubuntu WSL and it will be accessible from the Windows host.
+
+.NET is an open-source development platform from Microsoft that enables developers to build multi-platform applications from a single codebase.
 
 ## Requirements
 
 * A PC running Windows 11
 * The latest version of the WSL Windows Store application
-* Ubuntu, Ubuntu 22.04 LTS or Ubuntu Preview installed
+* Ubuntu 22.04 LTS, Ubuntu 24.04 LTS or Ubuntu Preview installed
```

Comment:
You can keep 'Ubuntu' (it points to the latest LTS) and remove 'Ubuntu Preview'. Now that systemd is enabled by default we don't need to point people to Preview. It is the development release and can be unstable. So for documentation, unless required, it is better to refer to stable versions.